requirements.adoc

```diff
@@ -188,6 +240,16 @@ the PACS to the imaging modality. The imaging modality will put the Study
 Instance UID in the images it creates. This allows the {project-short-name} to
 access the DICOM study and its images in the PACS via the Study Instance UID.
 
+```
+### Judy : Remember we have a Study , then series and then instance... so a CT Head is a study , then it can be axial , coronal or sagittal (the series) and each image in the series is an instance
+- Summarizing the above i think of the following functionality
+1. Ability to schedule an exam (this is in the orderable data where the date of examination is selected)
+2. Ability to create a modality worklist that can be queried by day
```

Comment:
**Modality Worklist**
modality worklist will be in the PACS not provided by this module for the foreseeable future. terminology is important here as this misguides peoples expectations of what the module does. Therefore I stated this at the top of the doc.

**Study**
at the moment we only have to Study in the module as a driver for the Study Instance UID as stated. This way we reference the Study in the PACS and embed a link in the module for you to open it. I dont think we should at least not now put the entire DICOM model into the module. So I would keep the Study slim only as a holder of reference to the Study in the PACS. Unless you have specific requirements.